app/Console/ReplayCommand.php

```diff
@@ -38,12 +38,17 @@ class ReplayCommand extends AbstractCommand
      * @var EventBusInterface
      */
     private $eventBus;
+    /**
+     * @var array
+     */
+    private $config;
 
-    public function __construct(CommandBusInterface $commandBus, EventStreamBuilder $eventStreamBuilder, EventBusInterface $eventBus)
+    public function __construct(CommandBusInterface $commandBus, EventStreamBuilder $eventStreamBuilder, EventBusInterface $eventBus, array $config)
```

Comment:
I don't like injecting the entire config array. It's usually a better idea to only inject the configuration variables you need.

Reply:
Hmm, it looks like we're using this config so we can write to it. Not optimal, but probably not something we should fix right now.

Reply:
Shouldn't it be passed by reference then?

Reply:
Good point. Will dive into it with @konrad126 tomorrow.

Reply:
You can see here how it got re-assigned to the global `$app['config']` after setting the properties. https://github.com/cultuurnet/udb3-silex/pull/456/files#diff-b729b0cc33328f7ab4412b9adc2d2a6fL238

That means there's indeed a bug here.
I would suggest to keep that part of the code as it was and not inject the configuration (not even by reference). When we refactor this towards a new container, chances are big we'll have a `Config` component that can override properties which we can then inject here.

Reply:
On the other hand, we can't remove `->getSilexApplication()` in that case. So maybe passing by reference is cleaner for now.

Reply:
Should be fixed

Reply:
Passing by reference didn't work:
```
PHP Notice:  Indirect modification of overloaded element of Silex\Application has no effect in /vagrant/source/udb3-silex/bin/udb3.php on line 79
```

Solved it by wrapping it in a `ConfigWriter` class for now